preload_parity_images.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eux
```

Comment:
Is this called automatically anywhere?

Reply:
not at the moment. @mikhaelsantos is in charge of the automatic image pushes to dockerhub.